package.json

```diff
@@ -31,5 +31,17 @@
         "eslint-config-airbnb-base": "^12.1.0",
         "eslint-plugin-import": "^2.9.0",
         "jest": "^22.4.2"
+    },
+    "resolutions": {
+        "mem": "^5.1.1",
+        "lodash": "^4.17.15",
+        "stringstream": "^1.0.0",
+        "js-yaml": "^3.13.1",
+        "just-extend": "^4.0.2",
+        "extend": "^2.0.2",
```

Comment:
I'm a bit confused - the existing `yarn.lock` already has `extend` in two versions: `2.0.2` and `3.0.1` (it's resolved to `2.0.2`. Is this because @KensoDev just fixed this?